### ISSUE TITLE
Change log

### ISSUE THREAD

**Issue by corydeppen** [IdentityModel/Thinktecture.IdentityModel#109]
Have you guys considered adding a `changelog.md` file to the repository to highlight what's changed with each release? Something simple based on [Keep a Changelog](http://keepachangelog.com/) would be helpful.

**Comment by leastprivilege** [comment]
Yep you are right - we gonna split identity model up into smaller chunks. That will be easier to follow.
